src/vs/workbench/workbench.web.main.ts

```diff
@@ -62,7 +62,7 @@ import 'vs/workbench/services/files/browser/elevatedFileService';
 import 'vs/workbench/services/workingCopy/browser/workingCopyHistoryService';
 import 'vs/workbench/services/userDataSync/browser/webUserDataSyncEnablementService';
 import 'vs/workbench/services/configurationResolver/browser/configurationResolverService';
-import 'vs/workbench/services/localization/browser/localeService';
+import 'vs/workbench/contrib/localization/browser/localeService';
```

Comment:
@TylerLeonhardt please move this down at the end of the file where contrib goes.

Reply:
Actually disregard this, how we typically do it is that there would be a `locale.contribution` file that declares the service. In other words, we do not delcare `contrib` services here but rather a main entry file.